MapleServer2/Database/SQL/Database.sql

```diff
@@ -723,6 +728,47 @@ CREATE TABLE `ugc_market_sales`
   DEFAULT CHARSET = utf8mb4
   COLLATE = utf8mb4_0900_ai_ci;
 
+--
+-- Table structure for table `mushking_royale_stats`
+--
+
+DROP TABLE IF EXISTS `mushking_royale_stats`;
+CREATE TABLE `mushking_royale_stats`
+(
+    `id`                               bigint       NOT NULL AUTO_INCREMENT,
```

Comment:
missed rename here

Reply:
that's stats. id should be id in stats. i made the rename changes on medals

Reply:
id should be ok.

Reply:
oh ok